src/Motion.js

```diff
@@ -217,6 +260,12 @@ function shouldStopAnimation (currentValues, values, currentVelocities) {
     // istanbul ignore if
     if (!Object.prototype.hasOwnProperty.call(values, key)) continue
 
+    if (isArray(values[key]) || isObject(values[key])) {
+      if (shouldStopAnimation(currentValues[key], values[key], currentVelocities[key])) {
```

Comment:
I initially wrote this with the condition flipped (and a `return false` after) because my thinking was: Not negating could end the animation cycle too early. However, after running the tests this was the only implementation that worked.

Does this seem logically correct to you?
Could you conceive a test case where this could still fail?